generatebundlefile/bundle.go

```diff
@@ -23,39 +23,35 @@ var (
 // +kubebuilder:object:generate=false
 type BundleGenerateOpt func(config *BundleGenerate)
 
-// Used for generating yaml for generating a new crd sample file
-func NewBundleGenerate(bundleName string, opts ...BundleGenerateOpt) *BundleNoStatus {
-	clusterConfig := &BundleNoStatus{
-		PackageBundle: &api.PackageBundle{
-			TypeMeta: metav1.TypeMeta{
-				Kind:       api.PackageBundleKind,
-				APIVersion: SchemeBuilder.GroupVersion.String(),
-			},
-			ObjectMeta: metav1.ObjectMeta{
-				Name:      bundleName,
-				Namespace: api.PackageNamespace,
-			},
-			Spec: api.PackageBundleSpec{
-				Packages: []api.BundlePackage{
-					{
-						Name: "sample-package",
-						Source: api.BundlePackageSource{
-							Registry:   "sample-Registry",
-							Repository: "sample-Repository",
-							Versions: []api.SourceVersion{
-								{
-									Name:   "v0.0",
-									Digest: "sha256:da25f5fdff88c259bb2ce7c0f1e9edddaf102dc4fb9cf5159ad6b902b5194e66",
-								},
+// Used for generating YAML for generating a new sample CRD file.
+func NewBundleGenerate(bundleName string, opts ...BundleGenerateOpt) *api.PackageBundle {
+	return &api.PackageBundle{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       api.PackageBundleKind,
+			APIVersion: SchemeBuilder.GroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      bundleName,
+			Namespace: api.PackageNamespace,
+		},
+		Spec: api.PackageBundleSpec{
+			Packages: []api.BundlePackage{
+				{
```

Comment:
opts isn't used here and why have sample-package I guess

Reply:
The sample package is because this function generates a sample bundle, which is one of the commands in the generatebundle command.